colossalai/inference/engine/policies/llama.py

```diff
@@ -20,20 +20,18 @@
 from ..modeling.llama import LlamaInferenceForwards
 
 try:
-    from colossalai.kernel.triton import rmsnorm_forward
-
+    from lightllm.models.llama.triton_kernel.rmsnorm import rmsnorm_forward as lightllm_rmsnorm_forward
     HAS_TRITON_RMSNORM = True
 except:
-    print("you should install triton from https://github.com/openai/triton")
+    print("Did not find rms-norm triton kernel")
+    print("You can use the following command to install: pip install git+https://github.com/ModelTC/lightllm.git@ece7b43f8a6dfa74027adc77c2c176cff28c76c8")
     HAS_TRITON_RMSNORM = False
 
 
 def get_triton_rmsnorm_forward():
     if HAS_TRITON_RMSNORM:
```

Comment:
This also looks weird to me. The function is to get triton rmsnorm forward, that means if triton is not found, an error should be raised, instead, the `HAS_TRITON_RMSNORM` check is put outside, and this will return None if triton rmsnorm is not found. This is absolutely against the normal expectation of a user running this function.

Reply:
if it returns none, the policy will not replace rmsnorm with triton rmsnorm of lightllm

Reply:
no, i mean this is not the correct to write this code.

Reply:
changed some logic.